File: BEE 1042.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var x = parseInt(line.shift());
var y = parseInt(line.shift());
var z = parseInt(line.shift());

var arr = [x, y, z];
arr.sort(function(a, b){return a - b});

console.log(arr[0]);
console.log(arr[1]);
console.log(arr[2]);
console.log();
console.log(x);
console.log(y);
console.log(z);

File: BEE 1042.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int x, y, z;
    cin >> x >> y >> z;
    int arr[] = {x, y, z};
    sort(arr, arr + 3);
    cout << arr[0] << endl;
    cout << arr[1] << endl;
    cout << arr[2] << endl;
    cout << endl;
    cout << x << endl;
    cout << y << endl;
    cout << z << endl;
    return 0;
}